server/controllers/finance/reports/priceList/index.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * @overview server/controllers/finance/reports/financial.employee.js
+ *
+ * @description
+ * This file contains code to create a PDF report for financial activities of an employee
+ *
+ * @requires Employee
+ * @requires ReportManager
+ */
+const _ = require('lodash');
+const ReportManager = require('../../../../lib/ReportManager');
+const db = require('../../../../lib/db');
+const util = require('../../../../lib/util');
+
+
+const TEMPLATE = './server/controllers/finance/reports/priceList/report.handlebars';
+
+const PDF_OPTIONS = {
+  filename : 'FORM.LABELS.PRICE_LIST',
+};
+
+exports.report = (req, res, next) => {
+  const options = req.query;
+
+  let report;
+
+  _.defaults(options, PDF_OPTIONS);
+
+  // set up the report with report manager
+  try {
+    report = new ReportManager(TEMPLATE, req.session, options);
+
+    const priceListUUID = db.bid(req.params.uuid);
+    return lookupPriceList(priceListUUID).then(data => {
+      const { items } = data;
+
+      items.forEach(item => {
+        if (item.priceListValue) {
+          if (item.is_percentage === 1) {
+            item.price += util.roundDecimal((item.price * item.priceListValue) / 100, 2);
+          } else {
+            item.price += util.roundDecimal(item.priceListValue, 2);
```

Comment:
I think you must fix this line with `=` instead of `+=` for not percentage values

Reply:
@mbayopanda last time you told me that if a price list is for example 100% and the inventory price is 0.5$, the patient will pay 0.5 + 0.5 = 1$, right?

Reply:
If the price isn't a percentage, we just replace the price.  That way the person responsible for the price list doesn't have to do a lot of calculations to try and arrive at his desired price.